pathfinder/tests/test_graphs.py

```diff
@@ -29,12 +29,14 @@ def test_routing_benchmark(
         toc = time.time()
         times.append(toc - tic)
     end = time.time()
-    for path in paths:
+    for path_object in paths:
+        path = path_object['path']
         fees = sum(G[node1][node2]['view'].fee for node1, node2 in zip(path[:-1], path[1:]))
         for node1, node2 in zip(path[:-1], path[1:]):
             view = G[node1][node2]['view']
             print('fee = ', view.fee, 'capacity = ', view.capacity)
         print('fee sum = ', fees)
+        print('pre-calculated fee sum = ', path_object['estimated_fee'])
```

Comment:
There will never be a difference between pre-calculated fee sum and fee_sum, so I'd remove line 34, and adapt line 37.